sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  is still missing. intel/llvm#14590 intel/llvm#14188 intel/lvm#12727
+  intel/llvm#14757 intel/llvm#13486 intel/llvm#13974 intel/llvm#13617
+- Enhanced compiler to annotate SYCL kernel arguments passed by value with
+  `__grid_constant__` for CUDA backend. intel/llvm#14322
+- Added initial support for sub-groups on Native CPU backend. intel/llvm#13979
+- Added support for `reqd_work_group_size` attribute to Native CPU backend.
+  intel/llvm#13175
+- Introduced some extra address space inference for `invoke_simd` API so that
+  backends are able to generate better code. intel/llvm#14628
+- Improved math built-ins support on Native CPU backend: added support for bf16
+  and pointers in generic address space. intel/llvm#13109 intel/llvm#13829
+  intel/llvm#13911 intel/llvm#13428 intel/llvm#13478 intel/llvm#13249
+  intel/llvm#13765
+- Improved debugging experience on Linux (CPU & GPU) and Windows (CPU AOT only).
+  intel/llvm#13107 intel/llvm#13938
+- Optimized device code linking process by providing device libraries in LLVM
+  IR format instead of fat object files which allowed us to skip unbundling
+  step. Note: FPGA path still uses fat object files.
+  intel/llvm#13604 intel/llvm#13693
+- Added missing lowering of `reqd_work_group_size` attribute for CUDA devices
+  so that device compiler can now see the attribute and use it during
+  compilation. intel/llvm#14502
+- Strengthened security-related compilation flags used to build libraries and
+  tools which are part of the intel/llvm SYCL implementation. intel/llvm#13327
+  intel/llvm#14135 intel/llvm#13800
+
+### SYCL Library
+
+- Added support for `sqrt` and `rsqrt` ESIMD function for `double` data type. intel/llvm#13254
+- Updated [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) extension implementation to support cubemap images. intel/llvm#12996
+- Added ESIMD API for dynamic allocation of named barriers. intel/llvm#13826
+- Updated [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) extension implementation to support sampled image arrays. intel/llvm#14237
+- Added implementation for whole graph update (`executable_command_graph::update`).
+  intel/llvm#13220 intel/llvm#14379 intel/llvm#14236
+- Added a warning about use of the deprecated `<CL/sycl.hpp>` header. intel/llvm#13569
+- Made `local_accessor::get_pointer` and `local_accessor::get_multi_ptr` throw
+  `invalid` exception if they are called on host. intel/llvm#13747
+- Extended detection of nested `queue` operations to support shortcut methods. intel/llvm#13659
+- Added overloads of various ESIMD APIs (`atomic_update`, `block_[load|store]`
+  and some other) which allow to omit some template arguments, thus simplifying
```

Comment:
```suggestion
  and some other) which allow the omission of some template arguments, thus simplifying
```

Reply:
Applied in 8b9264c7b212610675e3e16bdaf1d6af049c8cc1